packages/devtools-ui-kit/src/composables/notification.ts

```diff
@@ -0,0 +1,17 @@
+let _showNotification: typeof showNotification
+
+export type Position = 'top-left' | 'top-center' | 'top-right' | 'bottom-left' | 'bottom-center' | 'bottom-right'
+
+export function showNotification(data: {
```

Comment:
If we register them to auto-import, maybe we should prefix them as `devtoolsUiShowNotification`